### ISSUE TITLE
Type Error in compute_engine.py After Upgrading to v0.6.0

### ISSUE THREAD

**Issue by zabrewer** [krish-adi/barfi#6]
Love the new feature with block categorization - everything works as expected until you hit the Execute button.

To Re-Create:
1) Use the new base_blocks_category dict
2) Add at least 2 blocks
3) hit the execute button

Versions:
* Python 3.8
* barfi v0.6.0

relative code:
```
base_blocks = [logic_filter, to_dataframe, use_cached_data, save_to_cache, barchart_output, excel_output]
base_blocks_category = {
                        'Logic and Filter Blocks': [logic_filter],
                        'Data Conversion Blocks': [to_dataframe],
                        'Input Blocks':[use_cached_data],
                        'Output Blocks':[json_output, excel_output, barchart_output, save_to_cache]
                        }

barfi_result = st_barfi(
                base_blocks=base_blocks_category,
                compute_engine=True,
                load_schema=load_schema)
```

Error
```
  File "/Users/zach.brewer/Dev/pythonprojects/.virtualenvs/test-nodepython/lib/python3.8/site-packages/barfi/__init__.py", line 87, in st_barfi
    _ce._map_block_link()
  File "/Users/zach.brewer/Dev/pythonprojects/.virtualenvs/test-nodepython/lib/python3.8/site-packages/barfi/compute_engine.py", line 47, in _map_block_link
    _parent_block = next(
  File "/Users/zach.brewer/Dev/pythonprojects/.virtualenvs/test-nodepython/lib/python3.8/site-packages/barfi/compute_engine.py", line 48, in <genexpr>
    _b for _b in self._blocks if _b._type == _block['type'])
AttributeError: 'str' object has no attribute '_type'
```

**Comment by krish-adi** [comment]
Thanks for the bug report @zabrewer ! 

It's now fixed in v 0.6.1. You can update your package now.

**Comment by zabrewer** [comment]
Confirmed fixed!  Thanks!
